google/cloud/BUILD.bazel

```diff
@@ -157,3 +157,30 @@ load(":google_cloud_cpp_grpc_utils_benchmarks.bzl", "google_cloud_cpp_grpc_utils
         "@com_google_googleapis//google/spanner/v1:spanner_cc_grpc",
     ],
 ) for test in google_cloud_cpp_grpc_utils_benchmarks]
+
+load(":google_cloud_cpp_rest_internal.bzl", "google_cloud_cpp_rest_internal_hdrs", "google_cloud_cpp_rest_internal_srcs")
+
+cc_library(
+    name = "google_cloud_cpp_rest_internal",
```

Comment:
nit: the default visibility here is "public", consider changing the visibility of this target to "packages" or something similar.